app/editor/package.json

```diff
@@ -1,115 +1,117 @@
 {
-  "name": "tno-editor-app",
```

Comment:
linter

Reply:
Why is your linter doing this?  It makes it impossible to know what's going on.  Please find out when you can.

Reply:
Also you can highlight what changed = react-text-mask.

Reply:
Will try to figure out later today. But yes just the types for react-text-mask and the package itself